cukinia

```diff
@@ -196,6 +206,26 @@ _cukinia_http_request()
 	wget -q -O /dev/null "$url"
 }
 
+# _cukinia_mount: check for a specific mount point
+# arg1: the mount source
+# arg2: the mount destination
+# arg@: (optinal) a list of options to look for
+_cukinia_mount()
+{
+	local source="$1"; shift
+	local mount_point="$1"; shift
+	local re=""
+
+	for term in "$@"; do
+		re="$re$term.*"
+	done
+
+	re="$source.*on $mount_point.*$re"
```

Comment:
that will catch a lot of false positives. See above for a stricter awk-based approach.

Examples in real life:
  cukinia_mount /dev/sda1 / rw (which is common and legit)

Will generate re="/dev/sda1.*on /.*rw"
This will pass if there is a mount similar to this for example:
/dev/sda12 on /rw ro (which is common and legit)

And this sucks.